app/containers/List/ListItem.js

```diff
@@ -99,15 +120,21 @@ const ListItem = React.memo(({ ...props }) => {
 		return <Button {...props} />;
 	}
 	return (
-		<View style={{ backgroundColor: themes[props.theme].backgroundColor }}>
+		<View
+			style={{
+				backgroundColor:
+          props.backgroundColor || themes[props.theme].backgroundColor
```

Comment:
Can you destructure `backgroundColor` just to follow the same pattern as Button?